core/domain/state_domain.py

```diff
@@ -843,6 +843,37 @@ def from_dict(cls, written_translations_dict):
 
         return cls(translations_mapping)
 
+    def get_content_ids_that_are_correctly_translated(self, language_code):
+        """Returns a list of content ids in which translations are available in
```

Comment:
This description sounds like it's just describing "already translated". You should probably talk about the "needs update" part as well in order to justify the "correctly" part.

Reply:
Please change "translation are" ---> "translation is" (since 'translation' is singular).

Reply:
Done!